src/utils/treasury-utils.ts

```diff
@@ -50,7 +50,7 @@ export const getTreasuryDetails = async () => {
     id: config.treasuryAddress as string,
   }
 
-  const protocolDetailsPayloadn = {
+  const ConvexProtocolPayload = {
```

Comment:
why this is capital letter and everything else isnt?

Reply:
its now fixed and all vars are in camelcase